.github/workflows/pre-merge.yml

```diff
@@ -584,12 +586,12 @@ jobs:
         uses: actions/upload-artifact@ea165f8d65b6e75b540449e92b4886f43607fa02  # v4.6.2
         with:
           name: trivy-scan-report-${{ env.sanitized_image_name }}-${{ env.md5sum_value }}
-          path: trivy-${{ env.sanitized_image_name }}.txt
+          path: trivy-${{ env.sanitized_image_name }}.sarif
 
-      - name: Checkout code
-        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683  # v4.2.2
+      - name: Upload Trivy Results
+        uses: github/codeql-action/upload-sarif@28deaeda66b76a05916b6923827895f2b14ab387  # v3.28.16
```

Comment:
These artifacts will be uploaded in the `pre-merge` phase, that is, for some branch related to the PR.  
Do we know if they will still be visible in the Security tab after the PR is merged and the branch is removed?